tests/python/gaia-ui-tests/gaiatest/apps/system/app.py

```diff
@@ -36,9 +36,12 @@ def wait_for_status_bar_displayed(self):
         Wait(self.marionette).until(expected.element_displayed(*self._status_bar_locator))
 
     # A lot of tests, like mail or call received, need a longer timeout here
-    def wait_for_notification_toaster_displayed(self, timeout=30, message=None):
-        Wait(self.marionette, timeout).until(
-            expected.element_displayed(*self._notification_toaster_locator), message=message)
+    def wait_for_notification_toaster_displayed(self, timeout=30, message=None, appname=None):
+        el = self.marionette.find_element(*self._notification_toaster_locator)
+        Wait(self.marionette, timeout).until(expected.element_displayed(el), message=message)
+        if appname != None:
```

Comment:
`if appname is not None:` is a bit faster (we don't really need it here) and a bit more semantic.